tests.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 from api import create_app
+from api.mock_db import db
```

Comment:
As you broke down views into different files, consider doing the same thing for tests.